UI/MapWnd.cpp

```diff
@@ -1859,15 +1859,13 @@ void MapWnd::RenderFields() {
     glDisableClientState(GL_COLOR_ARRAY);
     glEnableClientState(GL_TEXTURE_COORD_ARRAY);
 
+    auto buffer_not_empty = [](const auto& field_buffer) { return !field_buffer.second.empty(); };
```

Comment:
can probably be `static constexpr`

Reply:
Actually thinking about it, now that I explicitly split the "visible" and "sensor ghost/not visible" fields into two separate collections, this check seems not necessary. The buffer should never be empty, as now, when it gets created, we do put vertices of at least one field into it

I kept the check from previous code, but it was necessary then, because texture had two buffers assigned in one collection at the same time, never mind if there were any visible/not visible fields of given texture. So you could end up with empty buffer here or there

Reply:
(I removed this check)